models/brac/useviews/useview_assessment.sql

```diff
@@ -15,10 +14,7 @@
     )
 }}
 
-SELECT
-{{ dbt_utils.surrogate_key(['reported', 'uuid']) }} AS useview_assessment_reported_age_uuid,
-*
-FROM(
+
```

Comment:
Is there any reason the 'rev_field' is being indexed?

Reply:
Good catch.